server/app/controllers/AddTask.scala

```diff
@@ -58,8 +58,8 @@ class AddTask @Inject()(executor: DynamicSuiteExecutor, dao: TaskDao, val messag
       },
       f => {
         def addTaskIfValid(traitName: String) = {
-          val testResultOpt = (r: Try[String]) => Option(service.testResult(r))
```

Comment:
@novakov-alexey this is first `testResult` and next line `testResult(testResult)` ?